workers/Dockerfile.base

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:16.04
+
+RUN \
+  apt-get update -qq && \
+  apt-get install -y \
+                     lsb-release && \
+  echo "deb http://archive.ubuntu.com/ubuntu $(lsb_release -sc) multiverse\n" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  echo "deb http://cran.cnr.berkeley.edu/bin/linux/ubuntu $(lsb_release -sc)/" \
+      >> /etc/apt/sources.list.d/added_repos.list && \
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9 && \
```

Comment:
Ok. Maybe a comment here (with a link to that R documentation) would be helpful. I guess maybe the comment would have to go above the "one-line" `RUN` statement.

`<rant>`Why doesn't Ubuntu offer a cleaner way to update R? Should we really be training users to install mysterious keys so they can install common software packages?`</rant>`

`<rant>`Okay, I wasn't actually done. I looked at the R documentation link you included. It says:

> Ubuntu archives on CRAN are signed with the key of “Michael Rutter marutter@gmail.com” with key ID E084DAB9

Really, r-project? REALLY? You couldn't come up with a more official key for distributing the software than "marutter@gmail.com"? So, if his Gmail gets hacked then everyone who has added this key is potentially susceptible to compromise?

Ok, now I'm really done.`</rant>`